src/test/java/org/folio/edge/orders/MainVerticleTest.java

```diff
@@ -458,4 +481,34 @@ public void testPlaceOrderTimeout(TestContext context) throws MalformedApiKeyExc
         new ErrorWrapper("REQUEST_TIMEOUT", MSG_REQUEST_TIMEOUT));
     assertEquals(respBody.toXml(), resp.body().asString());
   }
+
+  @Test
+  public void testShouldReturnEmptyResponseIfResponseFromVendorIsEmpty(TestContext context) throws JsonProcessingException {
+    String body = StringUtils.EMPTY;
+
+    final Response resp = RestAssured
+      .with()
+      .body(body)
+      .post("/orders?type=GOBI&apiKey=" + apiKey)
+      .then()
+      .statusCode(201)
+      .extract()
+      .response();
+
+    assertEquals(StringUtils.EMPTY, resp.body().asString());
+  }
+
+  @Test
+  public void testShouldCatchExceptionIfVendorResponsInvalide(TestContext context) throws JsonProcessingException {
```

Comment:
typo in test name "Invalide"

Reply:
Also, does this really throw JsonProcessingException?

Reply:
Thanks You are right. Tests was changed, but forgot change a name. Fixed